web/src/app/wizard/WizardScheduleForm.js

```diff
@@ -36,34 +37,24 @@ const styles = {
  */
 @withWidth()
 @withStyles(styles)
-export default class WizardScheduleForm extends React.Component {
-  static propTypes = {
-    onChange: p.func.isRequired,
-    value: valuePropType,
-    secondary: p.bool,
-  }
-
-  getKey = () => {
-    const { secondary } = this.props
+export const WizardScheduleForm = ({secondary, onChange, classes, secondary, value, width}) =>   {
```

Comment:
lets declare this as a function instead of a const:

```
export default function WizardScheduleForm({secondary, onChange, classes, secondary, value, width}) {
```

Reply:
also, secondary is defined twice here so we can delete one of the references.